x-pack/plugins/spaces/server/saved_objects/mappings.ts

```diff
@@ -19,18 +19,6 @@ export const SpacesSavedObjectMappings = deepFreeze({
         },
       },
     },
-    description: {
-      type: 'text',
-    },
-    initials: {
-      type: 'keyword',
-    },
-    color: {
-      type: 'keyword',
-    },
-    disabledFeatures: {
-      type: 'keyword',
-    },
     // This field isn't included in the mappings since it doesn't need to be indexed or searched, but it will still
     // appear in the `_source` field of the `space` document.
     // imageUrl: { type: 'text' },
```

Comment:
__nit:__ can we also move `imageUrl` to schemas and remove this comment?

Reply:
I added it to schemas in the latest commit, but why was it here as a comment to begin with?